pom.xml

```diff
@@ -610,6 +610,19 @@
                     </dependency>
                 </dependencies>
             </plugin>
+
+            <plugin>
+                <groupId>org.apache.maven.plugins</groupId>
+                <artifactId>maven-jar-plugin</artifactId>
```

Comment:
what is this for?

Reply:
To have the version populated in the jar manifest and available via `getClass().getPackage().getImplementationVersion())`